src/Components/Home/contribution/ContributionSection.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Text, View, Image } from 'react-native';
+import styles from '../styles';
+import SectionHeader from '../../SectionHeader';
+import ContributionCard from './ContributionCard';
+
+
+function ContributionSection() {
+    const contributionCards = [
```

Comment:
can we move text and pngs to content folder ?

Reply:
I can shift the text to the content folder , just wanted to understand why are we keeping all text in a separate folder? And all the other images are in the assets folder so should I still move it to content?

Reply:
So we can easily update content, without looking throw whole code